front/src/styles/scss/applications/stdcm/_card.scss

```
@@ -118,6 +118,10 @@
         margin-top: 14px;
       }
 
+      .loading-gauge {
+        margin-top: 14px;
+      }
```

We could replace this style and the previous one with this
```suggestion
      .towed-rolling-stock,
      .loading-gauge {
        margin-top: 14px;
      }
```

---

Done